modules/apps/portal-search-solr8/portal-search-solr8-impl/src/test/java/com/liferay/portal/search/solr8/internal/SolrIndexingFixture.java

```diff
@@ -257,7 +257,7 @@ protected NGramQueryBuilderImpl createNGramQueryBuilder() {
 			new NGramQueryBuilderImpl();
 
 		ReflectionTestUtil.setFieldValue(
-			nGramQueryBuilderImpl, "_nGramHolderBuilder",
+			nGramQueryBuilderImpl, "_nGramHolderBuilderSnapshot",
```

Comment:
@LisztLiszt This part is wrong. The field is now of type `Snapshot`, so you will fail to set a `NGramQueryBuilder` to it.

The correct way is to register the `nGramQueryBuilderImpl` using BundleContext. Please see Jiaxu's PR modifying ElasticSearch modules for example.